include/manager.hpp

```diff
@@ -127,15 +128,45 @@ class Manager
 
     /**
      * @brief Get expanded location code
-     * API can be used to get expanded location code for the given FRU inventory
-     * path.
+     * API to get expanded location code from the un-expanded location code.
      *
-     * @param[in] i_dbusObjPath - D-bus object path
+     * @param[in] i_locationCode - un-expanded location code.
+     * @param[in] i_nodeNumber - Denotes the node in case of a multi-node
+     * configuration, ignored on a single node system.
      *
-     * @return FRU's expanded Location code
+     * @return Location code of the FRU.
      */
-    std::string getExpandedLocationCode(
-        const sdbusplus::message::object_path& i_dbusObjPath);
+    std::string getExpandedLocationCode(const std::string& i_locationCode,
+                                        uint16_t i_nodeNumber);
+
+    /**
+     * @brief Get FRUS D-Bus object paths from expanded location code.
+     * An API to get list of FRU D-BUS object paths for a given expanded
+     * location code. Returns empty vector if no FRU found for that location
+     * code.
+     *
+     * @param[in] i_locationCode - Expanded location code.
+     *
+     * @return List of FRUs D-Bus object paths for the given location code.
+     */
+    types::ListOfPaths
+        getFRUsByExpandedLocationCode(const std::string i_locationCode);
```

Comment:
If it expects only expanded location code then lets rename it to  i_expandedLocationCode.

Reply:
Done